lib/components/util/dropdown.tsx

```diff
@@ -192,8 +195,25 @@ const Dropdown = ({
           {children}
         </DropdownMenu>
       )}
-    </DropdownContainer>
+    </DropdownContainerEl>
   )
 }
 
-export default Dropdown
+export const SortResultsDropdown = styled(Dropdown)`
+    position: relative;
+
+    ${DropdownButton} {
+      border-radius: 5px;
+      padding: 3px 7px;
+      min-width: 130px;
+      text-align: right;
```

Comment:
Why `text-align: right`? With the min width, this leaves unexplained padding to the left.